lesson08/assignment/bakers-home/src/components/PicView.vue

```diff
@@ -0,0 +1,53 @@
+<!-- picture component built with props and a mixin to add border -->
+<template>
+
+    <div class="carousel-item" >
+        <div class="d-flex justify-content-center">
+            <img 
+                :src="require('@/assets/' + picture)"  
+                class="d-block w-75 picture" 
+                :alt="altText" 
+                @click="toggleShow"
+                :class="{imgborder: isActive}"
+            >
+            <div class="carousel-caption d-none d-md-block">
+                <h5>{{ imgTitle }}</h5>
+            </div>
+        </div>
+    </div>
+</template>
+    
+<script>
+//toggle mixin
```

Comment:
You would probably want to move this mixin to its own file, maybe named mixins.js.  The point of a mixin is to have reusable code that you can import into any component.